docs/source/readout_noise.rst

```diff
@@ -0,0 +1,354 @@
+probabilities from a set of positive operators
+
+.. math::
+
+
+   E_{x'} := \sum_{x\in \mathcal{O}} p(x'|x)\Pi_x \ge 0
+
+that must sum to 1:
+
+.. math::
+
+
+   \sum_{x'\in\mathcal{O}'} E_x' = \sum_{x\in\mathcal{O}}\underbrace{\left[\sum_{x'\in\mathcal{O}'} p(x'|x)\right]}_{1}\Pi_x = \sum_{x\in\mathcal{O}}\Pi_x = 1.
+
+The above result is a type of generalized **Bayes' theorem** that is
+extremely useful for this type of (slightly) generalized measurement and
+the family of operators :math:`\{E_{x'}| x' \in \mathcal{O}'\}` whose
+expectations give the probabilities is called a **positive operator
+valued measure** (POVM). These operators are not generally orthogonal
+nor valid projection operators but they naturally arise in this
+scenario. This is not yet the most general type of measurement, but it
+will get us pretty far.
+
+How to model :math:`T_1` error
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+T1 type errors fall outside our framework so far as they involve a
+scenario in which the *quantum state itself* is corrupted during the
+measurement process in a way that potentially erases the pre-measurement
+information as opposed to a loss of purely classical information. The
+most appropriate framework for describing this is given by that of
+measurement instruments, but for the practical purpose of arriving at a
+relatively simple description, we propose describing this by a T1
+damping Kraus map followed by the noisy readout process as described
+above.
+
+Further reading
+~~~~~~~~~~~~~~~
+
+Chapter 3 of John Preskill's lecture notes
+http://www.theory.caltech.edu/people/preskill/ph229/notes/chap3.pdf
+
+How do I get started?
+---------------------
+
+1. Come up with a good guess for your readout noise parameters
+   :math:`p(0|0)` and :math:`p(1|1)`, the off-diagonals then follow from
+   the normalization of :math:`P_{x'|x}`. If your assignment fidelity
+   :math:`F` is given, and you assume that the classical bit flip noise
+   is roughly symmetric, then a good approximation is to set
+   :math:`p(0|0)=p(1|1)=F`.
+2. For your QUIL program ``p``, and a qubit index ``q`` call:
+
+   ::
+
+       p.define_noisy_readout(q, p00, p11)
+
+   where you should replace ``p00`` and ``p11`` with the assumed
+   probabilities.
+
+Estimate :math:`P_{x'|x}` yourself!
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can also run some simple experiments to estimate the assignment
+probability matrix directly from a QPU.
+
+**Scroll down for some examples!**
+
+.. code:: python
+
+    from __future__ import print_function, division
+    import numpy as np
+    import matplotlib.pyplot as plt
+    %matplotlib inline
+
+.. code:: python
+
+    from pyquil.quil import Program, MEASURE, Pragma
+    from pyquil.api.qvm import QVMConnection
+    from pyquil.gates import I, X, RX
+
+.. code:: python
+
+    DARK_TEAL = '#48737F'
+    FUSCHIA = '#D6619E'
+    BEIGE = '#EAE8C6'
+
+.. code:: python
+
+    cxn = QVMConnection()
```

Comment:
Maybe combine all the above setup code into one block? It takes up a lot of vertical space in the rendered docs